dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1188,15 +1206,19 @@ private String getAnExistingDOIForWorkflow(Workflow workflow) {
      * @param depositMetadata Metadata for the workflow version
      * @param depositionID Zenodo's ID for the deposition
      */
-    private void putDepositionOnZenodo(DepositsApi depositApi, DepositMetadata depositMetadata, int depositionID) {
+    private void putDepositionOnZenodo(DepositsApi depositApi, DepositMetadata depositMetadata,
+            int depositionID, File uploadFile) {
         NestedDepositMetadata nestedDepositMetadata = new NestedDepositMetadata();
         nestedDepositMetadata.setMetadata(depositMetadata);
         try {
             depositApi.putDeposit(depositionID, nestedDepositMetadata);
         } catch (ApiException e) {
             LOG.error("Could not put deposition metadata on Zenodo. Error is " + e.getMessage(), e);
-            throw new CustomWebApplicationException("Could not put deposition metadata on Zenodo." + " Error is " + e.getMessage(),
-                    HttpStatus.SC_BAD_REQUEST);
+            throw new CustomWebApplicationException("Could not put deposition metadata on Zenodo."
+                    + " Error is " + e.getMessage(), HttpStatus.SC_BAD_REQUEST);
```

Comment:
I think this should be a 5xx error, but not sure.

A 4xx typically  means  the client did something wrong. Do you know what causes this error? Is it something the client is doing wrong?

Reply:
changed to HttpStatus.SC_INTERNAL_SERVER_ERROR